packages/pluggableWidgets/web-view-native/src/WebView.xml

```diff
@@ -33,6 +46,10 @@
                 <caption>Open links externally</caption>
                 <description>If true, links embedded in source document will be opened in external browser instead of inside this widget.</description>
             </property>
+            <property key="useSharedCookies" type="boolean" defaultValue="false">
+                <caption>Use shared cookies</caption>
```

Comment:
Hello @henkjan-van-henten-mendix , I was trying to validate this scenario but I wasn't able to reproduce a valid case.I tried to set a script into HTML to create and check one cookie but doesn’t look correct for me. However, for the callback change I was able to test and validate it.

I saw you mention about the SSO using cookies in this case. 

Perhaps, would you have a suggestion to me test this scenario? Or perhaps an URL which you’re expecting to work with this changes. So, I can test and approve this PR.

Reply:
I’ve tried to recreate the issue earlier i had with the shared cookies. But i seems that cookies are already shared within the same app container (at least with iOS 14). So i also could not reproduce the need of the setting at this moment.

For now i removed the setting, I can always apply a new pull request if i do find the need for it later.

Reply:
Thanks!  😁